.docker/images/nginx/fastcgi.conf

```diff
@@ -0,0 +1,67 @@
+
+set $fastcgi_port "80";
+if ($http_x_forwarded_proto = 'https') {
+    set $fastcgi_https "on";
+    set $fastcgi_port  "443";
+}
+
+set_by_lua_block $remote_addr_clean {
+  if string.find(ngx.var.remote_addr, "^::ffff:") then
+    return string.match(ngx.var.remote_addr, "^::ffff:(.*)")
+  else
+    return ngx.var.remote_addr
+  end
+}
+
+# Allow downstreams to add fastcgi_params, usually any values included
+# this way may be overwritten by the default values below.
+include /etc/nginx/fastcgi/*.conf;
```

Comment:
This is added in https://github.com/uselagoon/lagoon-images/pull/995 and will let us customise the fastcgi params as we need.